python/opae.admin/opae/admin/fpga.py

```diff
@@ -574,17 +574,23 @@ def safe_rsu_boot(self, available_image, **kwargs):
         root_port = self.pci_node.root
         force = kwargs.pop('force', False)
         supports_dpc = root_port.supports_ecap('dpc')
-        if supports_dpc or force:
-            if supports_dpc:
-                self.log.debug('ECAP_DPC supported')
-            elif force:
-                self.log.warn('forcing rsu routine')
-            self.rsu_routine(available_image, **kwargs)
-        else:
-            if root_port.supports_ecap('aer'):
-                self.log.debug('ECAP_AER is supported')
+        if root_port.supports_ecap('aer'):
+            self.log.debug('ECAP_AER is supported')
+            if (not supports_dpc) or force:
```

Comment:
remove Round () brackets   if (not supports_dpc)
python code style

Reply:
thanks